scripts/package.sh

```diff
@@ -0,0 +1,57 @@
+#/bin/bash -e
+# Copyright 2014 Treode, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR=$(dirname $(readlink -f $0))
+
+echo "PACKAGING TREODE FOR Debian..."
+
+STAGEDIR0=/var/tmp/treode.$$
+STAGEDIR=${STAGEDIR0}/treode-0.2.0
+mkdir -p ${STAGEDIR}
+
+cp ${SCRIPT_DIR}/conf/treode.init ${STAGEDIR}
+cd ${STAGEDIR}
+
+wget https://oss.treode.com/examples/finagle/0.2.0/finagle-server-0.2.0.jar
```

Comment:
Shouldn't we get the .jar from the build? After build-merge script runs, it's available under examples/finagle/target